### PR TITLE
A bundle of fixes to the react tests

### PR DESCRIPTION
**Pull request by richvdh**
Most of these are about shutting up some warnings, but there is also a fix to
a flaky TimelinePanel test:

    The 'should load new events even if you are scrolled up' test was sometimes
    failing. It turned out that pagination *sometimes* wasn't starting soon
    enough after setting the scrollTop, and awaitPaginationCompletion was
    incorrectly believing it to have already been and gone.

    Add an awaitScroll to make sure that we wait long enough for the pagination
    to begin.

### CODE REVIEW

File: test/components/structures/TimelinePanel-test.js [hunk 1]
```diff
@@ -148,7 +152,7 @@ describe('TimelinePanel', function() {
             console.log("adding event");
 
             // a new event!
-            var ev = mkMessage();
+            var ev = mkMessage(31);
```
Comment (dbkr) [inline test/components/structures/TimelinePanel-test.js]: N_EVENTS + 1, maybe?

File: test/img/icon_context_message.svg [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+            </g>
+        </g>
+    </g>
+</svg>
```
Comment (dbkr) [inline test/img/icon_context_message.svg]: did you mean to commit this? can't see it referenced anywhere?

Comment (richvdh) [inline test/img/icon_context_message.svg]: Yeah, I did. It's referenced by EventTile, and having it here stops the browser logging a load of 404s. Obviously any image would do.

File: test/components/structures/TimelinePanel-test.js (hunk 1 above)
Comment (richvdh) [inline test/components/structures/TimelinePanel-test.js]: done

**Pull request merged**